samples/house_price_estimates/data/data_description.txt

```diff
@@ -0,0 +1,523 @@
+MSSubClass: Identifies the type of dwelling involved in the sale.	
```

Comment:
The directory contains a lot of files, with not clear purpose, e.g. `data_description.txt` where it is used ? 
Again, I'm not a KFP expert, sorry for maybe obvious questions.

Reply:
It's just a description of the data in the csv file.  Such descriptions are used by data scientists building models